src/schema/objects/enums.yaml

```diff
@@ -1405,3 +1405,24 @@ microvascular:
   display_name: microvascular
   description: |
     The origin of a tissue: microvascular
+postnatal:
+  value: postnatal
+  display_name: postnatal
+  description: |
+    This is the actual time that has passed since a subject's birth,
+    also known as “chronological age”.
+    If age_type is omitted, age is assumed to be this type.
+fetal:
+  value: fetal
+  display_name: fetal
+  description: |
+    Also known as "embryonic age," this measures the age of a
+    fetus or embryo from the time of conception. It's typically about two weeks
+    less than the gestational age.
```

Comment:
For humans, but not other species, right? Or is this common across a broad spectrum of species?

Reply:
If we want to be species independent here (which we should) we need to avoid time absolute times for life cycle stages.

"embryo stage" according to UBERON is defined as: "A life cycle stage that starts with fertilization and ends with the fully formed embryo." http://purl.obolibrary.org/obo/UBERON_0000068

Reply:
let's continue in #1839 on this fun activity